applications/ipc_radio/README.rst

```diff
@@ -47,7 +47,8 @@ The firmware supports the following development kits:
 
 .. table-from-sample-yaml::
 
-To automatically attach the firmware image, you need to use the :ref:`sysbuild`.
+To automatically attach the firmware image, you need to use the :ref:`zephyr:sysbuild`.
+To learn more about :ref:`zephyr:sysbuild` in the nRF Connect for VS Code extension see `nRF Connect Sysbuild`_.
```

Comment:
```suggestion
To learn more about :ref:`zephyr:sysbuild` in the |nRFVSC| see `nRF Connect Sysbuild`_.
```

Reply:
I'd say we don't need two links to the same page in two sentences next to one another:
```suggestion

The |nRFVSC| supports sysbuild during `build configuration setup <How to build an application_>`_.
To learn more about sysbuild in the extention, see `sysbuild support`_ page in its documentation.
```

I think this mention of nRF VSC makes more sense in Configuration or in Building and running as a single image...